### PR TITLE
Adding a utility to round TimeSpan instances and using it in tests where rounding was performed

### PR DESCRIPTION
**Pull request by fabiocav**
Adding a `TimeSpan` rounding extension as we're likely to need it in a couple of other spots. Updated code that would benefit from that logic to use it.

### CODE REVIEW

**Comment by fabiocav** [comment]
@paulbatum , based on the comments from your earlier PR. Figured this would come in handy.

File: test/WebJobs.Script.Tests/UtilityTests.cs [hunk 1]
```diff
@@ -72,7 +73,8 @@ public class UtilityTests
 
             // Workaround annoying test failures such as "Expected sw.Elapsed >= TimeSpan.FromSeconds(2); Actual: 1999.4092" by waiting slightly less than 2 seconds
             // Not sure what causes it, but either Task.Delay sometimes doesn't wait quite long enough or there is some clock skew.
-            Assert.True(sw.Elapsed >= TimeSpan.FromMilliseconds(1990), $"Expected sw.Elapsed >= TimeSpan.FromMilliseconds(1990); Actual: {sw.Elapsed.TotalMilliseconds}");
+            TimeSpan roundedElapsedSpan = sw.Elapsed.RoundSeconds(digits: 1);
+            Assert.True(roundedElapsedSpan >= TimeSpan.FromMilliseconds(2), $"Expected roundedElapsedSpan >= TimeSpan.FromMilliseconds(2); Actual: {roundedElapsedSpan.TotalMilliseconds}");
```
Comment (paulbatum) [inline test/WebJobs.Script.Tests/UtilityTests.cs]: As written, this test won't pass. I think you needed to change the comparison to use FromSeconds.

Comment (paulbatum) [reply test/WebJobs.Script.Tests/UtilityTests.cs]: Actually the test will pass but its not testing what is intended :)

Comment (fabiocav) [reply test/WebJobs.Script.Tests/UtilityTests.cs]: Oops... yeah, rushed through this and completely missed that this was now using milliseconds. Updated.

**Comment by fabiocav** [comment]
@paulbatum updated... will merge, but let me know if you have any comments.

**Pull request merged**